tutorials/flow/layout/01__nested_layouts/content.adoc

```diff
@@ -0,0 +1,276 @@
+= Nested Layouts in Flow
+
+:title: Nested Layouts in Flow
+:authors: sven ruppert
+:type: text, video
+:topic: layout
+:tags: layout, flow, nested-layout, v10
+:description: How to use Layouts in Flow
+:repo: https://github.com/vaadin-learning-center/flow-layout-nested-layouts
+:linkattrs:
+:imagesdir: ./images
+
+In this part we will have a look at the layout mechanism itself and how you could
+build more complex layouts structures.
+
+== The Hello World used for this example
+As "Hello World" basic project the *flow-helloworld-maven-meecrowave**
+from the learning center is used.
+You are able to find the latest version
+here: https://github.com/vaadin-learning-center/flow-helloworld-maven-meecrowave[https://github.com/vaadin-learning-center/flow-helloworld-maven-meecrowave]
+
+== The challenge
+How to define elements in your application that are needed everywhere
+like a MenuBar?
+
+=== Preparations for this tutorial
+First we are creating a view Views that we want to
+extend with some generic elements like a menubar.
+
+For this two classes are additionally created in this project.
+The names of this classes are *ViewA* and *ViewB*.
+In total we have three classes that are representing a View now.
+
+The classes are all extending the class **Composite<Div>** and the root element inside the Composite is
+a neutral div itself. If you want to read more about this, have a look at
+the documentation at
+
+https://vaadin.com/docs/v10/flow/creating-components/tutorial-component-composite.html [https://vaadin.com/docs/v10/flow/creating-components/tutorial-component-composite.html]
+
+Every View class will get an annotation *Route* for an individual path.
+For example the class *ViewA* will get a path called *ViewA*. To access this view you have to go
+to http://localhost:8080/ViewA[http://localhost:8080/ViewA]
+
+The same for the class called *ViewB*, the path will be *ViewB*
+
+.`*views/ViewA.java*`
+[source,java]
+----
+@Route(value = "ViewA")
+public class ViewA extends Composite<Div> {
+
+  public ViewA() {
+    getContent().add(new Span("ViewA"));
+  }
+}
+----
+
+The class itself will do nothing exciting. It will only write the view name itself on screen.
+For sure the navigation between the views is not very comfortable for the user.
+We need a menu bar, containing all links to all accessible views.
+
+
+=== the menu bar
+A good place for a menu bar is inside a layout, based on the idea that a layout is the generic structure of the
+ui. The menu bar itself is easy and straight forward.
```

Comment:
Avoid saying that things are easy, it has a risk of making people feel like they're dumb if they don't get it working.